ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionLoadingIndicator.swift

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+
+struct BraveVPNRegionLoadingIndicatorView: View {
+
+  @State
+  private var isAnimating = false
+
+  var body: some View {
+    ZStack {
+      Circle()
+        .stroke(lineWidth: 7)
+        .opacity(0.3)
+        .foregroundStyle(Color(braveSystemName: .iconInteractive))
+        .frame(width: 48, height: 48)
+      Circle()
+        .trim(from: 0.75, to: 1.0)
+        .stroke(style: StrokeStyle(lineWidth: 7, lineCap: .round))
+        .foregroundColor(Color(braveSystemName: .iconInteractive))
+        .frame(width: 48, height: 48)
+        .rotationEffect(Angle(degrees: isAnimating ? 360 : 0))
+        .onAppear {
+          withAnimation(Animation.linear(duration: 1.0).repeatForever(autoreverses: false)) {
+            isAnimating = true
+          }
+        }
+    }
+  }
+}
```

Comment:
Don't we already have this somewhere? Is `BraveCircularProgressViewStyle` not usable? If not, this should be a custom ProgressViewStyle (see Leo/AI implementation)

Reply:
BraveCircularProgressViewStyle is not the same I checked but it is amazing to know cause I was nt aware.

I created ProgressViewStyle as you asked at https://github.com/brave/brave-core/pull/24198/commits/83bc8e8bbc9b3fab1a23f025b53cf7e33dbbb46a